apps/next/src/pages/AccountManagement/components/ImportKeystoreFile/components/FileImage.tsx

```diff
@@ -0,0 +1,19 @@
+import { useTheme } from '@avalabs/k2-alpine';
+import DarkImage from '../assets/darkFileImage.svg';
+import LightImage from '../assets/lightFileImage.svg';
+
+const Light = () => {
+  return <img src={LightImage} alt="File Image" width={44} height={51} />;
+};
+
+const Dark = () => {
+  return <img src={DarkImage} alt="File Image" width={44} height={51} />;
+};
+
+export const FileImage = () => {
+  const theme = useTheme();
+
+  const image = theme.palette.mode === 'light' ? <Light /> : <Dark />;
+
+  return image;
```

Comment:
The file can be simplified even more: 
```suggestion
export const FileImage = () => {
  const theme = useTheme();

  const image = theme.palette.mode === 'light' ? LightImage : DarkImage

  return <img src={Image} alt="File Image" width={44} height={51} />;
```

This way you don't need to remember to sync width and height of multiple components